app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,13 @@
+class LikesController < ApplicationController
+  def create
+    post = Post.find(params[:post_id])
+    current_user = post.author
```

Comment:
- [ ] Same issue here, as discussed above. The current_user is still the first User in the app. Please use the current user as is.

Reply:
Same tenacity here.
The change has been mad here as well.

😄